plugins/ufm_consumer_plugin/build/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM ubuntu:20.04
+
+LABEL maintainer="atabachnik@nvidia.com"
+
+ARG PLUGIN_NAME="ufm_consumer"
+ARG DEBIAN_FRONTEND=noninteractive
+ARG BASE_PATH=/opt/ufm/ufm_plugin_${PLUGIN_NAME}
+ARG SRC_BASE_DIR=${PLUGIN_NAME}_plugin
+
+COPY ${SRC_BASE_DIR}/ ${BASE_PATH}/${SRC_BASE_DIR}/
+COPY utils/ ${BASE_PATH}/utils/
+
+COPY ${SRC_BASE_DIR}/conf/supervisord.conf /etc/supervisor/conf.d/
+COPY ${SRC_BASE_DIR}/conf/ufm_consumer_plugin.conf /etc/supervisor/conf.d/
+COPY ${SRC_BASE_DIR}/scripts/init.sh ${SRC_BASE_DIR}/scripts/deinit.sh ${SRC_BASE_DIR}/scripts/start_ufm.sh /
+
+
+RUN apt-get update && apt-get -y install supervisor python3.9 python3-pip vim curl sudo openssh-client bc acl iproute2 sshpass lftp zip rsync telnet qperf sqlite3 apache2 ssl-cert snmpd cron logrotate chrpath gawk libcairo2
```

Comment:
python is redundant here UFM have its own python inside the ufm-conda package

Reply:
removed